README.md

```diff
@@ -28,7 +28,7 @@ To update dependencies, run `composer update`.
 - TinyMCE Advanced
 - Yoast SEO
 
-## Compass
-This project uses [Compass](http://compass-style.org/) to compile the SASS files. To run the compiler:
+## Gulp
```

Comment:
Will you please make this match the [Running Project Locally](https://github.com/CSIS-iLab/csis-gulp-workflow#running-project-locally) and [Building for Production](https://github.com/CSIS-iLab/csis-gulp-workflow#building-for-production) examples in the Gulp Workflow repo?

https://github.com/CSIS-iLab/csis-gulp-workflow#building-for-production